root/static/scripts/edit/URLCleanup.js

```diff
@@ -4464,6 +4464,10 @@ const CLEANUPS: CleanupEntries = {
         /^(?:https?:\/\/)?(?:(?:www|mobile)\.)?twitter\.com(?:\/#!)?\//,
         'https://twitter.com/',
       );
+      url = url.replace(
+        /^(https:\/\/twitter\.com)\/intent\/user\?screen_name=([^\/?#]+)/,
```

Comment:
Is it worth handling an optional slash after /user?  Not sure if these are only linked from Twitter or other people can make them too.  (Interestingly, I don't get a popup with the slash added.)
```suggestion
        /^(https:\/\/twitter\.com)\/intent\/user\/?\?screen_name=([^\/?#]+)/,
```

Reply:
Since both work, we might as well.